cmd/captain/run.go

```diff
@@ -69,12 +69,7 @@ func createRunCmd(cliArgs *CliArgs) *cobra.Command {
 					case "github-step-summary":
 						stepSummaryPath := os.Getenv("GITHUB_STEP_SUMMARY")
 						if stepSummaryPath == "" {
-							return errors.WithDecoration(errors.NewConfigurationError(
-								"'github-step-summary' reporter misconfigured",
-								"The 'github-step-summary' reporter can only run within a GitHub Actions job where the "+
-									"'GITHUB_STEP_SUMMARY' environment variable is set.",
-								"",
-							))
+							continue
```

Comment:
Maybe we should write a debug log message here?

Reply:
So the logger is on the cli service, but it only requires the command so I think we could move that bit up higher?

Reply:
Seems reasonable to me but maybe @michaelglass or @pierrebeaucamp have thoughts

Reply:
Specifically, this change: https://github.com/rwx-research/captain/pull/26/commits/94f75d5c9334be0b945d9bdba0492a6d2ff4d6bd

Reply:
I'm for pushing the logger out of service (and ... I'd like to move service into the app config)

Reply:
> I'd like to move service into the app config

Can you elaborate on this?